app/controllers/actions/IdentifierAction.scala

```diff
@@ -144,6 +144,9 @@ class AuthenticatedIdentifierAction @Inject() (
       case None =>
         logger.warn(s"IdentifierAction: Agent without HMRC-AS-AGENT enrolment. Enrolments: $enrolments. Redirecting to /agent/use-agent-services")
         Future.successful(Left(Redirect(controllers.agent.routes.AgentUseAgentServicesController.onPageLoad)))
+      case _ =>
+        logger.warn("IdentifierAction: Agent does not have delegated authority for Client. Redirecting to /agent/client-not-identified")
+        Future.successful(Left(Redirect(controllers.client.routes.ProblemCBCIdController.onPageLoad)))
```

Comment:
In this case I don't think a `case _` should be added, we should just change the `case None` above into a `case _`